### ISSUE TITLE
Hovered item in WS window is not visible in high contrast

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3144]
Hovering over with mouse

![image](https://cloud.githubusercontent.com/assets/12820357/22716751/1abce346-ed4c-11e6-843e-9755cce67800.png)

**Comment by MikhailArkhipov** [comment]
https://github.com/Microsoft/RTVS/commit/953ce796947f6a11f334ab12f49cefbb30b382fa fixes this too

**Comment by jflam** [comment]
While using the white high contrast theme, this is what I see:

![image](https://cloud.githubusercontent.com/assets/8877/22951695/fa07fcf8-f2be-11e6-9327-4a269ffad133.png)

**Comment by MikhailArkhipov** [comment]
You have to restart VS. Not all colors are auto-picked when scheme changes. Generally themes do not change dynamically that often. If someone is using high contrast they typically set it once.
